comp/schedule.html

```diff
@@ -55,6 +55,7 @@ <h1>Match Schedule</h1>
 
 <h2>[[ session.description ]]</h2>
 
+<div class="schedule-container">
 <table class="schedule">
```

Comment:
Please can we indent the newly nested content? The style here is either to do that or to introduce more vertical space (and perhaps an HTML comment) between the layers of container. I don't mind which we do, though it would be good to do one or the other.